### ISSUE TITLE
Enlace a las FAQ

### ISSUE THREAD

**Issue by Peque** [python-spain/PyConES-2016#16]
He tenido dificultades para encontrar las preguntas frecuentes en la página. Creo que es una sección importante en un evento como este y que, por tanto, debería ser más fácilmente accesible (encontrar las FAQ en una entrada del blog creo que es un poco anti-intuitivo).

Tal vez podría ponerse una lista de enlaces relevantes en el pie de página (contacto, localización, FAQ...) o si no podría al menos añadirse una sección al final de "Información del evento" con un enlace a las preguntas frecuentes. Quizás esto sea lo más lógico: si después de leer la información del evento aún tienes preguntas sin resolver, entonces aquí tienes las más frecuentes.

Un saludo y felicidades por la página y por la organización! :blush:

**Comment by kikocorreoso** [comment]
Cuando hablas de las FAQ te refieres a esto: http://2016.es.pycon.org/en/blog/se-buscan-revisores-pycones-2016/

¿Has encontrado lo que necesitabas en las FAQ?

¿Incluirías algo más en las FAQ?

**Comment by astrojuanlu** [comment]
En realidad cada "cosa" tiene su FAQ: también el proceso de revisión. Igual hay que hacer una Meta-FAQ xD

**Comment by astrojuanlu** [comment]
Y por cierto @kikocorreoso, prueba de ello es que @Peque se refería a http://2016.es.pycon.org/es/blog/se-abre-la-llamada-a-propuestas-de-la-pycones-2016/

**Comment by kikocorreoso** [comment]
@Juanlu001 sí, creo que quería poner ese enlace en lugar del que he puesto. Quizá para los cosas más concretas (FAQ revisores) no habría que meterlo en unas FAQs generales de asistentes/ponentes pero sí que sería útil tener un FAQ general con información útil añadido a [esto](http://2016.es.pycon.org/en/info/).

**Comment by Peque** [comment]
@kikocorreoso Me refería a cualquier tipo de pregunta frecuente que pueda tener cualquier perfil asistente al evento o colaborador (es decir, yo creo que todas las FAQ deberían ser más fácilmente accesibles: FAQs de asistentes, de ponentes, de revisores...). No sé si tiene sentido ponerlas todas en una misma página (agrupadas por categorías), pero creo que sí estaría bien que estuviesen enlazadas desde la sección de "Información del evento" y posiblemente también desde algún enlace en el pie de página.

Sí, encontré lo que necesitaba en las FAQ. :blush:
